coreneuron/nrniv/nrnoptarg.cpp

```diff
@@ -372,7 +372,6 @@ static void graceful_exit(int err) {
 #if defined(nrnoptargtest)
 // for testing, compile with: g++ -g -I../.. nrnoptarg.cpp
 
-int nrnmpi_myid;
```

Comment:
This must exist in the context of a test as mentioned in the comment above it. If it is removed one should remove
the entire test. If not removed I guess it could be helpful to add a comment as to why it is there.

Reply:
@nrnhines: thanks Michael I overlooked this one. 
This is currently compiling correctly as I have moved the definition into nrnmpi.h. 
Nevertheless, I will move this file as a test case, which will clarify the intend of it.

Reply:
@nrnhines : Jeremy is working on namespace with c++ to make it as library. With c++, above definition result into duplicate definition error (but not with c). As we have tests for this in `tests/unit/cmdline_interface/test_cmdline_interface.cpp`, we can remove this from here.